### ISSUE TITLE
Invalid characters should be rejected rather than waiting for review

### ISSUE THREAD

**Issue by bardiharborow** [enwikipedia-acc/waca#495]
Is there a good reason why we allow invalid requests to be created in the first place?

**Comment by AfroThundr3007730** [comment]
I was just talking about this on IRC. We could make a set of regexes to filter requests that violate WP:UPOL and drop them, or at least flag them for quicker triage.

**Comment by stwalkerster** [comment]
Any automated filtering we do would have to be limited to purely what MediaWiki considers an invalid title, and be provable with extensive test cases to be sure of that.

There's the added complexity that what MediaWiki considers an invalid title is mostly determined by configuration.

Basically, it's more effort than it's worth. It's rare enough to see one of these anyway, and they're quick to deal with.

**Comment by Dane2007** [comment]
I agree, this is definitely more effort than it's probably worth based on the number of these requests we get.  If it ever were to be implemented, I would strongly lean towards just flagging it somehow but still requiring an ACC users interaction - i'm not a huge fan of auto dropping things.
